Content.Shared/Chemistry/Components/Solution.cs

```diff
@@ -68,6 +96,26 @@ public sealed partial class Solution : IEnumerable<ReagentQuantity>, ISerializat
         [DataField("temperature")]
         public float Temperature { get; set; } = 293.15f;
 
+        /// <summary>
+        ///     The total heat capacity of all reagents in the solution.
+        /// </summary>
+        [ViewVariables] 
+        public float HeatCapacity {get; private set;} = 0f;
+
```

Comment:
If you're going to start caching this then a test is required.

Reply:
the ``TestTemperatureCalculations`` test on ``Content.IntegrationTests\Tests\Chemistry\SolutionSystemTests.cs`` doesn't do that already?